src/middleware/notFound.middleware.ts

```diff
@@ -0,0 +1,8 @@
+import type { NextFunction, Request, Response } from 'express';
+
+const routeNotFound = (req: Request, res: Response, next: NextFunction) => {
+  console.log('route not found');
```

Comment:
As this not found error will send an message to the user so 
instead of logging the message we have to send a response too. 
Just send a bad request response and send a json message `Route does not exist`,